recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
     name: basemap
-    version: 1.0.8.dev
+    version: 1.0.8.dev0
```

Comment:
But I loved that `conda smithy rerender` got that error! I did not notice it and the binaries never made into the channel. (Which explains the Travis-CI error we got during the copy.)